src/main/target/common_defaults_post.h

```diff
@@ -422,6 +422,27 @@
 #ifndef SDCARD_SDIO_DMA_OPT
 #define SDCARD_SDIO_DMA_OPT (-1)
 #endif
+#ifndef SDIO_USE_4BIT
+#define SDIO_USE_4BIT true
```

Comment:
Seems to be less risky to default this to 1 bit.

Reply:
👍